Source/WebKit/UIProcess/Cocoa/_WKWarningView.mm

```diff
@@ -185,7 +185,7 @@ - (void)drawRect:(RectType)rect
     [colorForItem(WarningItem::BoxBackground, warning) set];
     auto square = CGRectMake(0, 0, exclamationPointSize, exclamationPointSize);
     [[UIBezierPath bezierPathWithRect:square] fill];
-    
+
```

Comment:
Style checker wasn't happy with this pre-existing whitespace. Same with the other similar changes.